packages/lesswrong/components/messaging/NewConversationButton.tsx

```diff
@@ -23,7 +25,8 @@ const NewConversationButton = ({ user, currentUser, children }: {
       data: {participantIds:[user._id, currentUser._id], ...alignmentFields},
     })
     const conversationId = response.data.createConversation.data._id
-    history.push({pathname: `/inbox/${conversationId}`})
+    const search = templateCommentId ? {search:`?${qs.stringify({templateCommentId: templateCommentId})}`} : {}
```

Comment:
Double check that this doesn't add `?templateCommentId=undefined` or something like that every time someone clicks the message button on someone's profile

Reply:
Oh yeah I did check that last time. It does not.